.pre-commit-config.yaml

```diff
@@ -50,7 +50,6 @@ repos:
     hooks:
       - id: pydoclint
         args: [
-          python/ray,
```

Comment:
what's the relationship between this change with resizing tests?

Reply:
hm bad merge; fixing